src/main/java/org/dave/compactmachines3/gui/psd/Page.java

```diff
@@ -33,10 +35,10 @@ public void draw(GuiPSDScreen guiPSDScreen, FontRenderer fontRenderer, RenderIte
 
     public void mouseClicked(GuiPSDScreen psd, int mouseX, int mouseY, int mouseButton) {
         if(mouseButton == 1) {
-            if(name == "welcome") {
+            if(name.equals("welcome")) {
```

Comment:
Should probably leave the `==` check here, and maybe consider moving the value to a const somewhere..

Reply:
`==` check?? It's a String....